.github/workflows/qa.yml

```diff
@@ -8,5 +8,7 @@ jobs:
       - uses: actions/checkout@master
       - name: PHPStan
         uses: "docker://oskarstark/phpstan-ga"
+        env:
+          REQUIRE_DEV: true
```

Comment:
@greg0ire, do you know if is this enough or is some change required at dev-kit?

Reply:
No change required on dev-kit, the github actions are not managed there yet.